index.html

```diff
@@ -28,44 +28,40 @@
     </header>
     
     <main>   
-    		<!-- Logo top left corner -->
-	    <div id="logo"> 
-		<img src="http://via.placeholder.com/150x150"> 
-	    </div> 
+    	    <!-- Logo top left corner -->
+	    <img id="logo"> 
+		    <img src="http://via.placeholder.com/150x150"> 
+        </img>
 
-		<!-- header/instructions Placeholder -->
-	    <div id="instructions" class="jumbotron">
-		<div id="jumboContainer" class="container">
-		  <h1 class="display-4"></h1>
-		  <p class="Plantinstructions"></p>
-		</div>
-	    </div>
+		    <!-- header/instructions Placeholder -->
```

Comment:
why is jumbotron in an article? Perhaps section would be a better replacement for div. I appreciate that you are replacing divs